### PR TITLE
Volume dashboard update

### PR DESCRIPTION
**Pull request by VikramAtMicrosoft**
Updated Readme with clear instructions for entitlements
Removed Power Apps part from readme as schema was outdated
Made edits in the code for volume dashboard
Updated the compiled code (.mez file)

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -183,365 +245,3 @@ After you get the records, from the transform tab convert data into a table. You
 
 ![](media/c809cf612f6589f202610125cb4d9fa9.png)
 
-# Connecting Power Apps and Power Automate to OSDU R2
```
Comment (Miller-Kyle) [inline Readme.md:186]: @VikramAtMicrosoft You should confirm with Kadri, if you haven't already, before you remove this section

File: Readme.md [hunk 2]
```diff
@@ -53,11 +57,67 @@ Callback URL needs to be added to the Authentication tab as a Web Redirect URI
 
 **search_api_url \<OSDU Host Here\>/api/search/v2/query**
 
-**data-partition-id opendes**
+**data-partition-id  \<Your Data Partiton ID Here\>**
 
 These parameters need to be provided by the deployment team for deployments in
 customer tenants
 
+## Authorization and Entitlements
```
Comment (Miller-Kyle) [inline Readme.md:65]: Instead of documenting how to add a user to an entitlement group, can you link to a community OSDU doc to do it?

File: Power BI Connector/OSDUWellsConnector/OSDUWellsConnector/OSDUWellsConnector.pq [hunk 3]
```diff
@@ -32,7 +32,7 @@ shared OSDUWellsConnector.Contents = (kind as text, query as text, optional limi
              ]  
            ))
     in
-        Source[results];
```
Comment (Miller-Kyle) [inline Power BI Connector/OSDUWellsConnector/OSDUWellsConnector/OSDUWellsConnector.pq:35]: Is this change required? It will break existing reports.

File: Readme.md (hunk 1 above)
Comment (VikramAtMicrosoft) [reply Readme.md:186]: Okay will do, I have removed the section because the schema is outdated in the present code and the steps will not work, so we do not want to ship that. I have confirmed with my mentor Krishna Nikhil, I will also confirm with Kadri

File: Power BI Connector/OSDUWellsConnector/OSDUWellsConnector/OSDUWellsConnector.pq (hunk 3 above)
Comment (VikramAtMicrosoft) [reply Power BI Connector/OSDUWellsConnector/OSDUWellsConnector/OSDUWellsConnector.pq:35]: Yes, this change is quite useful to retrieve counts for volume dashboards. Also I am not aware of any present customer usage and we can publish future templates based on the new version.

File: Readme.md (hunk 2 above)
Comment (VikramAtMicrosoft) [reply Readme.md:65]: I will add the link to the OSDU doc as well, but it was thought that having clear steps for entitlements in the connector readme itself would help customers setup the connector more smoothly.

File: Readme.md (hunk 1 above)
Comment (krisnik) [reply Readme.md:186]: Vikram, what we can try is - Move Power Apps setup to different file say, PowerApps-Setup.md and move this documentation to that. 

Whenever PowerApps issues are sorted, we can use the documentation

**Comment by VikramAtMicrosoft** [comment]
Added the postman files and also created a different readme for powerapps

**Review by krisnik — APPROVED** [review]
